globe/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no" />
+  <script src='https://unpkg.com/maplibre-gl@5.0.0-pre.1/dist/maplibre-gl.js'></script>
+  <link href='https://unpkg.com/maplibre-gl@5.0.0-pre.1/dist/maplibre-gl.css' rel='stylesheet' />
+  <style>
+    #map {position: absolute; top: 0; right: 0; bottom: 0; left: 0;}
+  </style>
+</head>
+<body>
+  <div id="map"></div>
+  <script>
+    var map = new maplibregl.Map({
+      container: 'map',
+      style: 'style.json',
```

Comment:
I would suggest using an existing style and only setting the protection...

Reply:
Is the recommended way to do this something like

```
var map = new ...

map.setProjection("globe");
```

?

Reply:
Or would you rather patch the style?

Reply:
Like Jakub does it in his demo with something like

```
fetch("https://demotiles.maplibre.org/style.json")
      .then(response => response.json())
      .then(fetchedStyle => {
				fetchedStyle["projection"] = { "type": "globe" };
				map = new maplibregl.Map({
					container: 'map',
					style: fetchedStyle,
					center: [0.0, 0.0],
					zoom: 3.0
				});
      });
```

Reply:
I think an API is missing here to solve this, that's why we should wait for an official release.

Reply:
Good point!